src/App/integrationTests/App.toggleOffline.test.js

```diff
@@ -45,18 +45,18 @@ test('Starting ONLINE - Toggle is checked and switched to OFFLINE, some navigati
 
   const offlineToggleSwitchTestIdAfterFirstClick = screen.getByTestId('offline-toggle-switch-test')
 
-  userEvent.click(offlineToggleSwitchTestIdAfterFirstClick)
+  await user.click(offlineToggleSwitchTestIdAfterFirstClick)
 
   await waitForElementToBeRemoved(() => screen.queryByLabelText('project pages loading indicator'))
 
   expect(await screen.findByTestId('offline-toggle-switch-label')).toHaveTextContent(
     "You're ONLINE",
   )
 
-  expect(within(sideNav).queryByRole('link', { name: 'Submitted' })).toBeInTheDocument()
-  expect(within(sideNav).queryByRole('link', { name: 'Project Info' })).toBeInTheDocument()
-  expect(within(sideNav).queryByRole('link', { name: 'Users' })).toBeInTheDocument()
-  expect(within(sideNav).queryByRole('link', { name: 'Data Sharing' })).toBeInTheDocument()
+  expect(within(sideNav).getByRole('link', { name: 'Submitted' })).toBeInTheDocument()
+  expect(within(sideNav).getByRole('link', { name: 'Project Info' })).toBeInTheDocument()
+  expect(within(sideNav).getByRole('link', { name: 'Users' })).toBeInTheDocument()
+  expect(within(sideNav).getByRole('link', { name: 'Data Sharing' })).toBeInTheDocument()
```

Comment:
The linter now flags more things! 👍

Reply:
A nice one it flags is when you forget the `await` that needs to go with the `waitFor`. Forgetting it can lead to false positives. 👍 👍

Reply:
nice!!